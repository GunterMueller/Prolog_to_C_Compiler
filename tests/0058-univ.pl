main :-
	[1,2] =.. ['.', 1, [2]],
	123 =.. [123],
	foo =.. [foo],
	X =.. [bar,x,Y], display(X), nl, X = bar(x, _),
	this(that, Z) =.. [NAME|ARGS],
	display(NAME), put(32), display(ARGS), nl,
	ARGS = [that, _], NAME = this, length(ARGS, 2).
