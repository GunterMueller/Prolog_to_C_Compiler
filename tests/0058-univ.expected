bar(x, _8)
this [that,_19]
